### PR TITLE
test: Ignore some problematic tests on powerpc and powerpc64*

### PR DESCRIPTION
**Pull request by glaubitz**
This updates the list of tests which can be safely ignored on powerpc and powerpc64*.

### CODE REVIEW

**Comment by glaubitz** [comment]
Note: I was unable to run the testsuite locally, the testsuite wouldn't get past the tidycheck script. I am therefore waiting for the CI to fail now so I can update the line numbers in the stderr files.

CC @shepmaster

File: src/test/codegen/x86_mmx.rs [hunk 1]
```diff
@@ -13,6 +13,9 @@
 // ignore-emscripten
 // ignore-mips
 // ignore-mips64
+// ignore-powerpc
```
Comment (shepmaster) [inline src/test/codegen/x86_mmx.rs]: this file is for x86, so this makes sense.

File: src/test/codegen/repr-transparent-aggregates-2.rs [hunk 2]
```diff
@@ -13,6 +13,9 @@
 // ignore-aarch64
 // ignore-asmjs
 // ignore-mips64
+// ignore-powerpc
```
[the remark this answers is not in the document]
Comment (glaubitz) [reply src/test/codegen/repr-transparent-aggregates-2.rs]: No idea. The test was added like that in 2be697bc215.

Maybe @rkruppe can shed some light here?

**Comment by glaubitz** [comment]
Could someone review this before the 1.27.0 release? @shepmaster ?

**Comment by glaubitz** [comment]
@rust-highfive r?

**Comment by glaubitz** [comment]
r?

**Comment by glaubitz** [comment]
Can someone pick this up, please? Otherwise rustc 1.27 might not build on Debian/ppc64. Thanks.

**Pull request merged**